common/perception_utils/test/src/test_object_classification.cpp

```diff
@@ -61,3 +61,160 @@ TEST(object_classification, test_getHighestProbLabel)
     EXPECT_EQ(label, ObjectClassification::CAR);
   }
 }
+
+// Test isVehicle
+TEST(object_classification, test_isVehicle)
+{
+
+  using autoware_auto_perception_msgs::msg::ObjectClassification;
+  using perception_utils::isVehicle;
+
+  {// True Case with uint8_t
+  std::vector<std::uint8_t> vehicle_labels;
+  vehicle_labels.push_back(ObjectClassification::BICYCLE);
+  vehicle_labels.push_back(ObjectClassification::BUS);
+  vehicle_labels.push_back(ObjectClassification::CAR);
+  vehicle_labels.push_back(ObjectClassification::MOTORCYCLE);
+  vehicle_labels.push_back(ObjectClassification::TRAILER);
+  vehicle_labels.push_back(ObjectClassification::TRUCK);
+  
+  for(auto label: vehicle_labels){
+    EXPECT_TRUE(isVehicle(label));
+  }
+  }
+
+  // False Case with uint8_t
+  {
+  std::vector<std::uint8_t> non_vehicle_labels;
+  non_vehicle_labels.push_back(ObjectClassification::UNKNOWN);
+  non_vehicle_labels.push_back(ObjectClassification::PEDESTRIAN);
+  for(auto label: non_vehicle_labels){
+    EXPECT_FALSE(isVehicle(label));
+  }
+  }
+
+  // True Case with object_classifications
+  {  // normal case
+    std::vector<autoware_auto_perception_msgs::msg::ObjectClassification> classification;
+    classification.push_back(createObjectClassification(ObjectClassification::CAR, 0.5));
+    classification.push_back(createObjectClassification(ObjectClassification::TRUCK, 0.8));
+    classification.push_back(createObjectClassification(ObjectClassification::BUS, 0.7));
+    EXPECT_TRUE(isVehicle(classification));
+  }
+
+  // False Case with object_classifications
+  {  // false case
+    std::vector<autoware_auto_perception_msgs::msg::ObjectClassification> classification;
+    classification.push_back(createObjectClassification(ObjectClassification::PEDESTRIAN, 0.8));
+    classification.push_back(createObjectClassification(ObjectClassification::BICYCLE, 0.7));
+    EXPECT_FALSE(isVehicle(classification));
+  }
+
+} // TEST isVehicle
+
+
+// TEST isCarLikeVehicle
+TEST(object_classification, test_isCarLikeVehicle)
+{
+
+  using autoware_auto_perception_msgs::msg::ObjectClassification;
+  using perception_utils::isCarLikeVehicle;
+
+  {// True Case with uint8_t
+  std::vector<std::uint8_t> carlike_labels;
+  carlike_labels.push_back(ObjectClassification::BUS);
+  carlike_labels.push_back(ObjectClassification::CAR);
+  carlike_labels.push_back(ObjectClassification::TRAILER);
+  carlike_labels.push_back(ObjectClassification::TRUCK);
+  
+  for(auto label: carlike_labels){
+    EXPECT_TRUE(isCarLikeVehicle(label));
+  }
```

Comment:
Is there any reason to make a variable of `carlike_labels`?

EXPECT_TRUE(isCarLikeVehicle(ObjectClassification::BUS)); looks better to me.

Reply:
Thanks. I agree with you and fixed it.
At that time, I just thought it's better to use for loop so that I can prevent writing too many EXPECT_TRUE state.